e2e/cypress/integration/guest_accounts/member_invitation_ui_spec.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+ * Note: This test requires Enterprise license to be uploaded
+ */
+import {getRandomInt} from '../../utils';
+
+describe('Guest Account - Member Invitation Flow', () => {
+    before(() => {
+        // # Enable Guest Account Settings
+        cy.apiUpdateConfig({
+            GuestAccountsSettings: {
+                Enable: true,
+            },
+        });
+
+        // # Login as "sysadmin" and go to /
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+    });
+
+    it('MM-18039 Verify UI Elements of Members Invitation Flow', () => {
+        // # Get Current Team Name
+        let teamName = '';
+        const email = `temp-${getRandomInt(9999).toString()}@mattermost.com`;
+        cy.get('#headerTeamName').should('be.visible').invoke('text').then(((text) => {
+            teamName = text.trim();
+        }));
+
+        // #Open Invite People
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+        cy.get('#invitePeople').should('be.visible').click();
+
+        // *Verify UI Elements in initial step
+        cy.getByTestId('invitationModal').within(($el) => {
+            cy.wrap($el).find('h1').should('have.text', `Invite people to ${teamName}`);
+        });
+        cy.getByTestId('inviteMembersLink').should('be.visible').within(($el) => {
+            cy.wrap($el).getByTestId('inviteMembersSection').should('have.text', 'Invite MembersInvite new team members with a link or by email. Team members have access to messages and files in open teams and public channels.');
+            cy.wrap($el).find('.arrow').click();
+        });
+
+        // *Verify Share Link field
+        cy.getByTestId('shareLinkInput').should('be.visible').and('not.have.value', '');
```

Comment:
I think we should actually test the value of the text, like:
```suggestion
        cy.getByTestId('shareLinkInput').should('be.visible').and('have.value', `${baseUrl}/signup_user_complete/?id=${inviteId}`);
```
Where:
- `baseUrl` from Cypress.config('baseUrl');`
- `inviteId` from getting team by Id.  May add new API custom command of `cy.apiGetTeam`
```javascript
/**
 * Gets a team on the system
 * * @param {String} teamId - The team ID to get
 * All parameter required
 */

Cypress.Commands.add('apiGetTeam', (teamId) => {
    return cy.request({
        headers: {'X-Requested-With': 'XMLHttpRequest'},
        url: `api/v4/teams/${teamId}`,
        method: 'GET',
    }).then((response) => {
        expect(response.status).to.equal(200);
        return cy.wrap(response);
    });
});
```

Reply:
Changes done.